content/docs/command-reference/import.md

```diff
@@ -103,6 +103,11 @@ repo at `url`) are not supported.
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
+- `--no-exec` - create `.dvc` file without actually downloading `url`. E.g. if
+  the file or directory already exist it can be used to skip download.
+  `dvc commit <out>.dvc` should be used to calculate the URL and data hash,
+  update the `.dvc` files, and save existing data to the cache.
```

Comment:
```suggestion
- `--no-exec` - create a `.dvc` file without actually downloading `url`. E.g. if
  the file or directory already exist, this option can be used to skip the download.
  `dvc commit <out>.dvc` should be used to update the import `.dvc` files, and save existing data to the cache.
```

Reply:
BTW we should apply similar copy edits to `import-url --no-exec` — I'll do it separately 🙂

Reply:
Btw, it should be not `url` in this case I think. `W/o downloading data or file` or something like that.

Reply:
Updated.